src/main/java/bio/terra/cli/command/groups/List.java

```diff
@@ -16,15 +17,19 @@
   /** List the groups to which the current user belongs. */
   @Override
   protected void execute() {
-    java.util.List<ManagedGroupMembershipEntry> groups =
-        new SamService(Context.getServer(), Context.requireUser()).listGroups();
-    formatOption.printReturnValue(groups, List::printText);
+    java.util.List<Group> groups = Group.list();
+    formatOption.printReturnValue(
+        groups.stream()
+            .sorted(Comparator.comparing(Group::getName))
+            .map(group -> new UFGroup(group))
+            .collect(Collectors.toList()),
+        List::printText);
   }
 
   /** Print this command's output in text format. */
-  private static void printText(java.util.List<ManagedGroupMembershipEntry> returnValue) {
-    for (ManagedGroupMembershipEntry group : returnValue) {
-      OUT.println(group.getGroupName());
+  private static void printText(java.util.List<UFGroup> returnValue) {
+    for (UFGroup group : returnValue) {
+      group.print();
```

Comment:
do you need a separator or something between them?

Reply:
No, they'll all print on different lines, with indentation if they need >1 line.